lib/webview/package.json

```diff
@@ -4,18 +4,20 @@
   "private": true,
   "type": "module",
   "devDependencies": {
-    "@types/vscode-webview": "1.57.0",
+    "@types/prismjs": "1.26.0",
     "@types/react": "18.0.26",
     "@types/react-dom": "18.0.10",
-    "eslint-plugin-react": "^7.32.1",
-    "eslint": "^8.32.0"
+    "@types/vscode-webview": "1.57.0",
+    "eslint": "^8.32.0",
+    "eslint-plugin-react": "^7.32.1"
   },
   "dependencies": {
     "@rubberduck/common": "*",
+    "prismjs": "1.29.0",
+    "prop-types": "15.8.1",
```

Comment:
Prop-types might have been a requirement of the old diff viewer. Maybe it can be removed.